backend/search/id.txt

```diff
@@ -0,0 +1 @@
+0
```

Comment:
Contrary to media backup (which is run on a different host), I'd suggest to add "indexed" flag to the `content_source` table. The backup script can fetch first X non-indexed records, index them and update status in the database.

Additionally, the indexing script can take a parameter that clears "indexed" flag as the first step (to allow reindexing from scratch).

Rationale for that:
1. In the future we'd want to switch to realtime indexing
2. storing `id.txt` in git introduces sync and update problems, let's avoid that